samples/grids/data-grid/cell-activation/Pages/DataGridCellActivation.razor

```diff
@@ -0,0 +1,40 @@
+﻿@page "/DataGridCellActivation"
+@* this namespace is required for Infragistics controls *@
+@using Infragistics.Blazor.Controls
+@inject IInfragisticsBlazor InfragisticsBlazor;
+@inject IJSRuntime JSRuntime;
+
+<div class="igContainer">
+    
+    <div class="igComponent">
+
+        @if (DataSource != null)
+        {            
+            <DataGrid Height="100%" Width="100%"
+                      DataSource="@DataSource"
+                      AutoGenerateColumns="false"
+                      SelectionMode="GridSelectionMode.SingleCell"
+                      ActivationMode="GridActivationMode.Cell">
+                <TextColumn Field="Name" Width="170" />
+                <TextColumn Field="Street" HeaderText="Address" Width="150"/>
+                <TextColumn Field="City" Width="130"/>
+                <DateTimeColumn Field="Birthday" Width="150"/>               
```

Comment:
add 3-4 more columns to fill our some white space (see other samples that use EmployeeData)